### PR TITLE
Feature/vdx 261 switch env

### PR DESCRIPTION
**Pull request by sanderPostma**

### CODE REVIEW

File: packages/oid4vci-demo-frontend/package.json [hunk 1]
```diff
@@ -40,6 +39,7 @@
     "@types/node": "^18.18.3",
     "@types/react": "^18.2.23",
     "@types/react-dom": "^18.2.7",
+    "process": "^0.11.10",
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/package.json]: why do we need this? we were already using process without this package

File: packages/oid4vci-demo-frontend/src/components/SSICardView/index.tsx [hunk 2]
```diff
@@ -1,13 +1,11 @@
-import React from 'react';
+import React, {useState} from 'react'
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/src/components/SSICardView/index.tsx]: unused import

File: packages/oid4vci-demo-frontend/src/components/SSIPrimaryButton/index.tsx [hunk 3]
```diff
@@ -1,9 +1,6 @@
-import React from 'react';
-import {
-    getCurrentEcosystemComponentConfig,
-    
-    SSIPrimaryButtonConfig
-} from "../../ecosystem-config"
+import React, {useState} from 'react'
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/src/components/SSIPrimaryButton/index.tsx]: useState is unused

File: packages/oid4vci-demo-frontend/src/ecosystem/ecosystem.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import {
+    assertRoutes,
+    ComponentConfig,
+    EcosystemGeneralConfig,
+    getEcosystemRootConfig,
+    getEcosystemRoutes,
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/src/ecosystem/ecosystem.ts]: unused import

File: packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx [hunk 5]
```diff
@@ -25,17 +20,18 @@ type State = {
...
 const SSICredentialIssueRequestPage: React.FC = () => {
     const location = useLocation();
+    const ecosystem = useEcosystem()
     const flowRouter = useFlowRouter<SSICredentialIssueRequestPageConfig>()
-    const config = flowRouter.getPageConfig()
-    const generalConfig: EcosystemGeneralConfig = getCurrentEcosystemGeneralConfig();
-    const buttonConfig = getCurrentEcosystemComponentConfig('SSISecondaryButton') as SSISecondaryButtonConfig;
+    const pageConfig = flowRouter.getPageConfig()
+    const generalConfig = ecosystem.getGeneralConfig()
+    const buttonConfig = ecosystem.getComponentConfig<SSISecondaryButtonConfig>('SSISecondaryButton')
```
Comment (BtencateSphereon) [inline packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx]: unused

**Review by BtencateSphereon — CHANGES_REQUESTED** [review]
mostly beauty remarks and a question why we need a process package. i think i need to see a functional demo to fully finish my review

File: packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx (hunk 5 above)
Comment (sanderPostma) [reply packages/oid4vci-demo-frontend/src/pages/SSICredentialIssueRequestPage/index.tsx]: That one was there already. fixed in next commit

File: packages/oid4vci-demo-frontend/src/components/SSICardView/index.tsx (hunk 2 above)
Comment (sanderPostma) [reply packages/oid4vci-demo-frontend/src/components/SSICardView/index.tsx]: Ran import optimization on components

File: packages/oid4vci-demo-frontend/src/components/SSIPrimaryButton/index.tsx (hunk 3 above)
Comment (sanderPostma) [reply packages/oid4vci-demo-frontend/src/components/SSIPrimaryButton/index.tsx]: Ran import optimization

File: packages/oid4vci-demo-frontend/src/ecosystem/ecosystem.ts (hunk 4 above)
Comment (sanderPostma) [reply packages/oid4vci-demo-frontend/src/ecosystem/ecosystem.ts]: Ran import optimization

**Review by BtencateSphereon — APPROVED** [review]
LGTM

**Pull request merged**